net/wget/Makefile

```diff
@@ -72,7 +72,8 @@ CONFIGURE_ARGS+= \
 	--without-libpsl
 
 CONFIGURE_VARS += \
-	ac_cv_header_uuid_uuid_h=no
+	ac_cv_header_uuid_uuid_h=no \
+	ac_cv_libssl=yes
```

Comment:
Should this be inside the ssl BUILD_VARIANT section?

Reply:
Yes it should. Fixed.